### PR TITLE
[Python] Python interface for creating threadblock graphs

### PR DESCRIPTION
**Pull request by jiazhihao**

### CODE REVIEW

File: src/threadblock/element_binary.cc [hunk 1]
```diff
@@ -21,27 +21,27 @@ namespace mirage {
...
-  TBOperator *op =
-      create_elementbinary_op(input1, input2, mirage::type::TB_MUL_OP);
-  assert(op != nullptr);
-  operators.push_back(op);
-  return op->output_tensors[0];
+  return elementbinary(input1, input2, mirage::type::TB_MUL_OP);
+}
+
+STensor *Graph::mul(STensor const *input1, STensor const *input2) {
+  return elementbinary(input1, input2, mirage::type::TB_ADD_OP);
```
Comment (interestingLSY) [inline src/threadblock/element_binary.cc]: typo?

File: src/threadblock/element_binary.cc [hunk 2]
```diff
@@ -21,27 +21,27 @@ namespace mirage {
...
-  TBOperator *op =
-      create_elementbinary_op(input1, input2, mirage::type::TB_DIV_OP);
-  assert(op != nullptr);
-  operators.push_back(op);
-  return op->output_tensors[0];
+  return elementbinary(input1, input2, mirage::type::TB_DIV_OP);
+}
+
+STensor *Graph::div(STensor const *input1, STensor const *input2) {
+  return elementbinary(input1, input2, mirage::type::TB_ADD_OP);
```
Comment (interestingLSY) [inline src/threadblock/element_binary.cc]: typo?

**Review by interestingLSY — CHANGES_REQUESTED** [review]
There seems to be some typos in the code

File: src/threadblock/element_binary.cc (hunk 1 above)
Comment (jiazhihao) [reply src/threadblock/element_binary.cc]: Fixed.

File: src/threadblock/element_binary.cc (hunk 2 above)
Comment (jiazhihao) [reply src/threadblock/element_binary.cc]: Fixed

**Comment by jiazhihao** [comment]
Good catch. The typos should have been fixed.

**Review by interestingLSY — APPROVED** [review]

**Pull request merged**